src/test/OptionSettlement.t.sol

```diff
@@ -349,40 +583,37 @@ contract OptionSettlementTest is Test, NFTreceiver {
                 expiryTimestamp: testExpiryTimestamp,
                 exerciseAsset: WETH_A,
                 underlyingAmount: testUnderlyingAmount,
-                settlementSeed: 1234567,
+                settlementSeed: 0,
                 exerciseAmount: testExerciseAmount,
                 nextClaimId: 1
             })
         );
     }
 
-    function testEvent_write() public {
+    function testEventWrite() public {
         uint256 expectedFeeAccruedAmount = ((testUnderlyingAmount / 10_000) * engine.feeBps());
 
         vm.expectEmit(true, true, true, true);
         emit FeeAccrued(WETH_A, ALICE, expectedFeeAccruedAmount);
 
-        vm.expectEmit(true, true, true, false); // ignore data for now (TODO calculate claimId)
+        vm.expectEmit(true, true, true, false);
```

Comment:
Unrelated to your PR — we could keep this TODO, bc this event assertion is not checking the data (everything after the 3 topics, in this case the claim ID)

Reply:
Ignore, I see you moved to issue #74